code/FindShirt.js

```diff
@@ -10,13 +9,13 @@ exports.function = function(searchTerm, style, brand, gender, minPrice, maxPrice
     records = lib.findShirts(SHIRTS, searchTerm)
   }
   if (brand) {
-    records = records.filter(function (record) { return textLib.fuzzyMatch(record.brand, brand)})
+    records = records.filter(function (record) { return record.brand.toLowerCase().includes(brand.toString().trim().toLowerCase())})
   }
   if (style) {
-    records = records.filter(function (record) { return textLib.fuzzyMatch(record.style, style)})
+    records = records.filter(function (record) { return record.style.toLowerCase().includes(style.toString().toLowerCase())})
```

Comment:
how come you use trim() above and not here?

Reply:
trim() is used on brand name which don't have spaces in them
> brand: "RalphLauren",
> brand: "TommyHilfiger",

so if someone were to search for "Ralph Lauren" it wouldn't match our saved "RaplphLauren"
all other properties have spaces in them so individual words can just be matched.

Reply:
That's not what trim does

https://www.w3schools.com/jsref/jsref_trim_string.asp

Reply:
ah.. oops. thanks

Reply:
I'm taking this out. These paths are never used! all training examples annotate with searchTerm but searchTerm is only ever matched against title. I'll put in match on all of these into the function in utils.js